SCF1/item.java

```diff
@@ -0,0 +1,219 @@
+package assignment1;
+
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class item 
```

Comment:
1. Please include class level multi-line comment.
2. Class name should be Capitalise.